### PR TITLE
Sprint 1 merged

### PR DESCRIPTION
**Pull request by MDunning14**
Merge the changes up to the Milestone 2 demo into Main

### CODE REVIEW

**Review by abaileys — APPROVED** [review]

File: harkify/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.helloworld">
```
Comment (willywil548) [inline harkify/android/app/src/main/AndroidManifest.xml]: Probably should update the namespace

File: harkify/lib/notedetails.dart [hunk 2]
```diff
@@ -1,109 +1,165 @@
...
+                              Container(
+                                margin: const EdgeInsets.only(
+                                  top: 10,
+                                ),
+                                child: Center(
+                                  child: Text(
+                                    //passed date String should display here - Alec
+                                    dateFormat
+                                        .format(selectedNote.data.dateTime),
+                                    // dateString[0] + ":" + dateString[1],
```
Comment (willywil548) [inline harkify/lib/notedetails.dart]: Looks like this isn't needed anymore

File: harkify/lib/recordnote.dart [hunk 3]
```diff
@@ -9,8 +12,13 @@ class SpeechScreen extends StatefulWidget {
 
 class _SpeechScreenState extends State<SpeechScreen> {
   final SpeechToText _speech = SpeechToText();
+  //final VoiceHelper _helper = VoiceHelper();
```
Comment (willywil548) [inline harkify/lib/recordnote.dart]: Some leftover testing code?

File: harkify/lib/textnoteservice.dart [hunk 4]
```diff
@@ -1,50 +1,116 @@
...
+
+        final notesDirectory =
+            fileSystem.directory('${docsDirectory.path}/harkify');
+        notesDirectory.createSync();
+        return notesDirectory;
+      }
+    } catch (MissingPluginException) {}
+
+    // With unit tests, use a temp folder
+    //docsDirectory = await fileSystem.systemTempDirectory.createTemp('harkify');
```
Comment (willywil548) [inline harkify/lib/textnoteservice.dart]: More leftover testing code?

**Review by willywil548 — CHANGES_REQUESTED** [review]
Please update the namespace before merge

File: harkify/android/app/build.gradle [hunk 5]
```diff
@@ -36,7 +36,7 @@ android {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.example.helloworld"
```
Comment (willywil548) [inline harkify/android/app/build.gradle]: application id isn't really hello world is it?

**Review by abaileys — APPROVED** [review]

**Review by abaileys — APPROVED** [review]

**Review by willywil548 — COMMENTED** [review]
LGTM, Please resolve the Lint erros

**Comment by MDunning14** [comment]
Too many changes to track here, we're closing this and starting over

**Pull request closed without merging**